packages/dx-react-demos/src/demo-sources/chart-basic/bootstrap3/bar.skip.jsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/jsx-filename-extension */
+import * as React from 'react';
+import {
+  Chart,
+  ArgumentAxis,
+  ValueAxis,
+  BarSeries,
+} from '@devexpress/dx-react-chart-svg';
+import { contributors as data } from '../../../demo-data/data-vizualization';
+
+export default class Demo extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: data.slice(0, 7),
+      width: 700,
+      height: 400,
+      axes: [{ name: 'commits', min: 0 }, { name: 'login', type: 'band' }],
+      series: [
+        {
+          valueField: 'contributions',
+          argumentField: 'login',
+          axisName: 'commits',
+          name: 'BarSeries',
+        },
+      ],
+    };
+  }
+  render() {
+    return (
+      <Chart
+        data={this.state.data}
```

Comment:
Usually, we [decompose](https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-react-demos/src/demo-sources/grid-basic/column-alignment.jsxt#L35) the `state` object in the `render` method.